cmd/seed.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/uwblueprint/shoe-project/internal/database"
+	"github.com/uwblueprint/shoe-project/internal/database/migrations"
+	"go.uber.org/zap"
+)
+
+var (
+	seedCmd = &cobra.Command{
```

Comment:
Do we not want to restrict this to running in DEV mode only? (Like we did before by checking config?)

Reply:
yeah more I think about it, we should! We can just show an error message if done without `dev` mode